modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// storage/logical-volume-manager-storage.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-upgrading-lvms-on-sno_{context}"]
+= Upgrading logical volume manager storage on {sno} clusters
+
+Currently, it is not possible to upgrade from OpenShift Data Foundation Logical Volume Manager Operator 4.11 to logical volume manager storage 4.12 on {sno} clusters.
+
+You need to perform the following steps:
+
+. Back up any data that you want to preserve on the PVCs.
+. Delete all PVCs provisioned by the OpenShift Data Foundation Logical Volume Manager Operator and their pods.
+. Reinstall the logical volume manager storage on {product-title} 4.12.
+. Recreate the workloads.
+
+[NOTE]
+====
+Ensure that you back up your data and copy it to the PVCs created after upgrading to 4.12 because the data will not be preserved during this process.
```

Comment:
"Ensure that you back up your data" seems like it happens first, in step 1. Should this be a final step about copying it back to the PVCs after upgrading? And maybe move the "the data will not be preserved during this process" warning up to before they do the upgrade?

Reply:
Good point. I'm wondering if recreating the workloads and copying the data back to the PVCs is the same thing?

> . Back up any data that you want to preserve on the persistent volume claims (PVCs).
. Delete all PVCs provisioned by the {rh-storage} Logical Volume Manager Operator and their pods.
. Reinstall the logical volume manager storage on {product-title} 4.12.
. Recreate the workloads.
. Copy the backup data to the PVCs created after upgrading to 4.12.

@brandisher @suleymanakbas91 Could you please take a look? :)